packages/opal-common/opal_common/fetcher/providers/http_fetch_provider.py

```diff
@@ -48,6 +50,7 @@ class Config:
 class HttpFetchEvent(FetchEvent):
     fetcher: str = "HttpFetchProvider"
     config: HttpFetcherConfig = None
+    client_type: Literal["httpx", "aiohttp"] = "httpx"
```

Comment:
@omer9564 you might want to include the client_type within the config object
otherwise i am not sure there is an easy way to pass it via the network.

Reply:
Actually a better way would be to add an environment variable to the PDP to control the preferred http client instead. That would allow us to release a non breaking change.

Reply:
changed